deploy/rbac-permissions-operator-config/05-dedicated-admins-aggregate-cluster.ClusterRole.yaml

```diff
@@ -363,4 +363,16 @@ rules:
   - list
   - watch
 ### END
+### BEGIN - customer can view observe tab data in console
+### Permission needed in project openshift-monitoring
+- apiGroups:
+  - monitoring.coreos.com
+  resourceNames:
+  - k8s
+  resources:
+  - prometheuses/api
+  verbs:
+  - get
+### END - customer can view observe tab data in console
+
```

Comment:
Changes to this role are not needed.  The osd-readers ClusterRole aggregates to dedicated-readers which in turn aggregates to dedicated-admins.